powerstore/datasource_host_group.go

```diff
@@ -0,0 +1,267 @@
+						"id": schema.StringAttribute{
+							Description:         "Unique identifier of the host group.",
+							MarkdownDescription: "Unique identifier of the host group.",
+							Computed:            true,
+						},
+						"name": schema.StringAttribute{
+							Description:         "Host group name.",
+							MarkdownDescription: "Host group name.",
+							Computed:            true,
+						},
+						"description": schema.StringAttribute{
+							Description:         "Host group description.",
+							MarkdownDescription: "Host group description.",
+							Computed:            true,
+						},
+						"host_connectivity": schema.StringAttribute{
+							Description:         "Connectivity type for hosts and host groups.",
+							MarkdownDescription: "Connectivity type for hosts and host groups.",
+							Computed:            true,
+						},
+						"host_connectivity_l10n": schema.StringAttribute{
+							Description:         "Localized message string corresponding to host_connectivity",
+							MarkdownDescription: "Localized message string corresponding to host_connectivity",
+							Computed:            true,
+						},
+						"hosts": schema.ListNestedAttribute{
+							Description:         "Properties of a host.",
+							MarkdownDescription: "Properties of a host.",
+							Computed:            true,
+							NestedObject: schema.NestedAttributeObject{
+								Attributes: map[string]schema.Attribute{
+									"id": schema.StringAttribute{
+										Description:         "Unique identifier of the host.",
+										MarkdownDescription: "Unique identifier of the host.",
+										Computed:            true,
+									},
+									"name": schema.StringAttribute{
+										Description:         "The host name.",
+										MarkdownDescription: "The host name.",
+										Computed:            true,
+									},
+									"host_group_id": schema.StringAttribute{
+										Description:         "Associated host group, if host is part of host group.",
+										MarkdownDescription: "Associated host group, if host is part of host group.",
+										Computed:            true,
+									},
+								},
+							},
+						},
+						"mapped_host_groups": schema.ListNestedAttribute{
+							Description:         "Details about a configured host or host group attached to a volume.",
+							MarkdownDescription: "Details about a configured host or host group attached to a volume.",
+							Computed:            true,
+							NestedObject: schema.NestedAttributeObject{
+								Attributes: map[string]schema.Attribute{
+									"id": schema.StringAttribute{
+										Description:         "Unique identifier of a mapping between a host and a volume.",
+										MarkdownDescription: "Unique identifier of a mapping between a host and a volume.",
+										Computed:            true,
+									},
+									"host_id": schema.StringAttribute{
+										Description:         "Unique identifier of a host attached to a volume.",
```

Comment:
As discussed, please have a look if we need to add volume details here rather than host_id and host_group_id

Reply:
added volume_id and volume_name